examples/stm32/xively_demo_stm32f4xx_nucleo_wifi/Src/demo_io.c

```diff
@@ -0,0 +1,358 @@
+#include <assert.h>
```

Comment:
Sorry, I misunderstood your comments. Yeah, this file is the interface I wrote so we can keep all GPIO code away from `main.c`